docs/excel/excel-add-ins-ranges-set-get.md

```diff
@@ -47,6 +47,95 @@ Excel.run(function (context) {
 }).catch(errorHandlerFunction);
 ```
 
+## Select the edge of a used range (online-only)
+
+> [!NOTE]
+> The `Range.getRangeEdge` and `Range.getExtendedRange` methods are currently only available in ExcelApiOnline 1.1. To learn more, see [Excel JavaScript API online-only requirement set](../reference/requirement-sets/excel-api-online-requirement-set.md).
+
+The [Range.getRangeEdge](/javascript/api/excel/excel.range#getRangeEdge_direction__activeCell_) and [Range.getExtendedRange](/javascript/api/excel/excel.range#getExtendedRange_directionString__activeCell_) methods let your add-in replicate the behavior of the keyboard selection shortcuts, selecting the edge of the used used range based on the currently selected range. See [Get used range](excel-add-ins-ranges-get.md#get-used-range) to learn more about used ranges.
```

Comment:
```suggestion
The [Range.getRangeEdge](/javascript/api/excel/excel.range#getRangeEdge_direction__activeCell_) and [Range.getExtendedRange](/javascript/api/excel/excel.range#getExtendedRange_directionString__activeCell_) methods let your add-in replicate the behavior of the keyboard selection shortcuts, selecting the edge of the used range based on the currently selected range. To learn more about used ranges, see [Get used range](excel-add-ins-ranges-get.md#get-used-range).
```
I think the modifier phrase "based on the currently selected range" implies that the UsedRange is in use. "Used used" just doesn't sound right.

Reply:
Oops, that "used used" was a typo. Good catch!